### PR TITLE
Added Vips version 7 back as its own formulae.

### PR DESCRIPTION
**Pull request by skunkworker**
I'm currently using a ruby vips library in development that has not been updated to use version 8 yet and doesn't like newer versions. I created a separate vips7 formulae to preserve backwards compatibility.

### CODE REVIEW

File: vips7.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  homepage "http://www.vips.ecs.soton.ac.uk/"
+  url "http://www.vips.ecs.soton.ac.uk/supported/7.42/vips-7.42.3.tar.gz"
+  sha256 "6d001480b75a20d04d44869fb4cb93e4203e73ecd865dc68b1c11f56c9e74baa"
+
+  bottle do
+    root_url "https://homebrew.bintray.com/bottles-science"
+    sha256 "277998919346d9192b49e9cf77938d83ed5965c1e082e4bc609eaf459178e7ce" => :yosemite
+    sha256 "4102d4f73fc6a2f3463ee6701183b544c3775104e6df481567b200f1d016302b" => :mavericks
+    sha256 "e2b090c7561349998d3a162a375821534bc0656eb4266bff281ff451e4cee4ff" => :mountain_lion
+  end
```
Comment (iml) [inline vips7.rb]: Please remove the bottle block; it is configured by the test-bot.

**Comment by dpo** [comment]
It doesn't look like anything in Homebrew/Science depends on `vips`. Should `vips7` really be in here, or could it be sufficient for you to maintain it in a temporary tap until your library upgrades to version 8?

**Comment by skunkworker** [comment]
@dpo I use the vips 7 for the https://github.com/jcupitt/ruby-vips project.

**Comment by sjackman** [comment]
http://bot.brew.sh/job/Homebrew%20Science%20Pull%20Requests/2258/version=yosemite/console
```
==> brew style vips7

1 file inspected, no offenses detected
Error: 2 problems in 1 formula

==> audit problems
homebrew/science/vips7:
 * File should end with a newline
 * Formula should have a desc (Description).
```

**Comment by jcupitt** [comment]
vips8 is supposed to support the full vips7 API, it should work and it's a bug if it doesn't. Can you explain the issues you were seeing?

It seems to work for me with current brew:

```
$ brew install vips 
==> Installing vips from homebrew/homebrew-science
==> Downloading https://homebrew.bintray.com/bottles-science/vips-8.1.0.el_capit
######################################################################## 100.0%
==> Pouring vips-8.1.0.el_capitan.bottle.tar.gz
🍺  /usr/local/Cellar/vips/8.1.0: 177 files, 12M
$ gem install ruby-vips
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/universal-darwin15/rbconfig.rb:213: warning: Insecure world writable dir /usr/local in PATH, mode 040777
Fetching: ruby-vips-0.3.9.gem (100%)
Building native extensions.  This could take a while...
Successfully installed ruby-vips-0.3.9
Parsing documentation for ruby-vips-0.3.9
unable to convert "\xCA" from ASCII-8BIT to UTF-8 for ext/header.o, skipping
...
Installing ri documentation for ruby-vips-0.3.9
1 gem installed
```

If possible I'd like to get rid of the vips7 formula. It's installing an old version (7.42) with known (and fixed) bugs.

https://github.com/jcupitt/libvips/blob/master/ChangeLog

**Comment by skunkworker** [comment]
@jcupitt I'll take a look at it

**Comment by skunkworker** [comment]
@jcupitt I tried to reproduce the earlier issues I was having but I was unable too. I thought there was an issue with libxml2 not being installed, but the new 8.1.0 worked just fine. I'm fine in getting rid of the vips7 formula.

**Comment by jcupitt** [comment]
OK great. I've made a pull request to remove `vips7.rb`. Thanks for testing @skunkworker

**Pull request closed without merging**